docker-compose-dev.yml

```diff
@@ -0,0 +1,39 @@
+version: "2"
```

Comment:
I'm not sure this very specific local file for development should be included here. Maybe we could move it to the readme for those who want to debug it? Buddy isn't just about Kafka, so I think the contents of this file could be placed somewhere in the readme about how to debug the application in general. 📝